pkg/services/playlist/playlistimpl/xorm_store.go

```diff
@@ -180,6 +181,51 @@ func (s *sqlStore) List(ctx context.Context, query *playlist.GetPlaylistsQuery)
 	return playlists, err
 }
 
+func (s *sqlStore) ListAll(ctx context.Context, orgId int64) ([]playlist.PlaylistDTO, error) {
+	db := s.db.GetSqlxSession() // OK because dates are numbers!
```

Comment:
I gave up trying to make the queries work in xorm!  too much magic is required, and I don't see any downside here